Taskfile.yaml

```diff
@@ -47,6 +47,7 @@ tasks:
     desc: Configure repository from Ansible vars
     prompt: Any conflicting config in the root kubernetes and ansible directories will be overwritten... continue?
     dir: "{{.BOOTSTRAP_DIR}}"
-    cmd: ansible-playbook configure.yaml
+    cmds:
+      - ../.venv/bin/ansible-playbook configure.yaml
```

Comment:
is this change intended?

Reply:
Ah yeah. I didn't catch this. The env _should_ contain everything needed for `ansible-playbook` to be found (at least it works on my machine). Also`cmd` was introduced in a recent taskfile update. Maybe @brunnels had some other ideas behind this change.

Reply:
If you just run `task configure` and haven't activated the env you will get an error message.  This fixes that.  Same thing is already being done here https://github.com/onedr0p/flux-cluster-template/blob/main/.taskfiles/AnsibleTasks.yaml#L27

Reply:
Good point! I would make this line be `cmd: ../.venv/bin/ansible-playbook configure.yaml` then

Reply:
code will be updated in my next push

Reply:
You could also remove `dir: "{{.BOOTSTRAP_DIR}}"` and update the cmd to match (might be best)

Reply:
```cmd: ansible-playbook {{.BOOTSTRAP_DIR}}/configure.yaml```

Reply:
Sure, will be in next push

Reply:
Have you considered setting a path variable instead?

Reply:
I wish that would work, but it's and issue upstream with Taskfile: https://github.com/go-task/task/issues/482

Reply:
In fact I am doing that, but it doesn't work and should be commented out with a `# TODO`:
https://github.com/onedr0p/flux-cluster-template/blob/main/.taskfiles/AnsibleTasks.yaml#L6